storage/mempool.go

```diff
@@ -185,6 +185,45 @@ func (s *Storage) GetPendingUserState(stateID uint32) (*models.UserState, error)
 	}, nil
 }
 
+// does not return the full list of states, only the pending c2ts
+func (s *Storage) GetPendingUserStates(pubkey *models.PublicKey) ([]models.UserState, error) {
+	result := make([]models.UserState, 0)
+
+	maxUint32 := ^uint32(0)
+	pubKeyID := maxUint32
+
+	accounts, err := s.AccountTree.Leaves(pubkey)
```

Comment:
Does the accounttree have pending state?

Reply:
it does not, the account tree is the batched state!